@hts-forking/src/index.js

```diff
@@ -168,6 +168,25 @@ async function getHtsStorageAt(address, requestedSlot, blockNumber, mirrorNodeCl
         );
     }
 
+    // Encoded `address(tokenId).isAssociated()` slot
+    // slot(256) = `isAssociated`selector(32) + padding(192) + accountId(32)
+    if (
+        nrequestedSlot >> 32n ===
+        0x4d8fdd6d_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000n
+    ) {
+        const accountId = `0.0.${parseInt(requestedSlot.slice(-8), 16)}`;
+        const { tokens } = (await mirrorNodeClient.getTokenRelationship(accountId, tokenId)) ?? {
+            tokens: [],
+        };
+        if (tokens.some(t => t.token_id === tokenId)) {
+            return ret(
+                `0x${Number(1).toString(16).padStart(64, '0')}`,
```

Comment:
nit: can we use `toIntHex256`? 

in `main` there is a new version of this function that accepts `number`.